apartment/views.py

```diff
@@ -1,3 +1,207 @@
-from django.shortcuts import render
+import json, re
 
-# Create your views here.
\ No newline at end of file
+from haversine import haversine
+from datetime  import datetime
+
+from django.views     import View
+from django.http      import JsonResponse
+from django.db.models import Q, Max, Min, Avg
+
+from .models          import Apartment, ApartmentComplex, District, Neighborhood, TradeType
+
+class ComplexDetailView(View):
+    def get(self, request, complex_id):
+        try:
+            apartment_complex = ApartmentComplex.objects.prefetch_related('apartment_set').get(id=complex_id)
+            apartments        = apartment_complex.apartment_set.all()
+
+            context = {
+                'complex_id'      : complex_id,
+                'complex_name'    : apartment_complex.name,
+                'completion_year' : apartment_complex.completion_year,
+                'image_url'       : apartment_complex.image_url,
+                'household_number': apartment_complex.household_number,
+                'address'         : apartment_complex.address,
+                'size'            : [
+                    {
+                        'm^2'     : size.area,
```

Comment:
```suggestion
                        'square_decametre '     : size.area,
```
키값이지만 이런 선언은 지양해주세요.